deployment/ansible/roles/model-my-watershed.app/tasks/reverse-proxy.yml

```diff
@@ -1,4 +1,13 @@
 ---
+- name: Ensure Nginx cache directory exists
+  file: path="{{ nginx_cache_dir }}"
+        state=directory
+        owner="{{ ansible_ssh_user }}"
```

Comment:
Ideally, this directory would be owned by `www-data`. I think `mode=0775` makes it work in this case though.

Also just a little worried about how this will impact other things going on in the root volume, but the lowest entry point for ephemeral storage is an `m3.medium`. We're currently running `t2` instances everywhere else.